aggie-coding-club/Rev-Registration
# Select all section select

## Description

It adds a select/deselect all button to section select.

## How to test

For any term and class, click the "section" button and you should see the button.

## Screenshots

New Version:
![myGif](https://user-images.githubusercontent.com/49422137/101994368-8d12c080-3c87-11eb-92b9-4b9779aa01c4.gif)

## Related tasks

Closes #363
